code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -437,3 +437,37 @@
 		chaser.monster_damage_boost = FALSE // Weaker cuz no cooldown
 		chaser.damage = 20
 		log_combat(user, target, "fired a chaser at", src)
+
+/obj/item/crusher_trophy/ice_demon_cube
+	name = "demonic cube"
+	desc = "A stone cold cube dropped from an ice demon."
+	icon_state = "ice_demon_cube"
+	///how many will we summon?
+	var/summon_amount = 2
+	///cooldown to summon demons upon the target
+	COOLDOWN_DECLARE(summon_cooldown)
+
+/obj/item/crusher_trophy/ice_demon_cube/effect_desc()
+	return "mark detonation to unleash demonic ice clones upon the target!"
```

Comment:
```suggestion
	return "mark detonation to unleash demonic ice clones upon the target"
```

this text is either shown in a sentence going like _Causes `effect_desc()` when attached to a kinetic crusher._ or _Causes `effect_desc()`._ which makes this exclamation point out of place

Reply:
on another note, another mob source for crusher damage that is going untracked (until my crusher pr hopefully) 😔

Reply:
i have tendancy to be overdramatic so  i throw exclamation marks everywhere